packages/frontend/core/src/modules/docs-search/entities/docs-indexer.ts

```diff
@@ -36,7 +36,7 @@ export class DocsIndexer extends Entity {
   /**
    * increase this number to re-index all docs
    */
-  static INDEXER_VERSION = 6;
+  static INDEXER_VERSION = 10;
 
   private readonly jobQueue: JobQueue<IndexerJobPayload> =
```

Comment:
[](https://support.github.com/contact/report-abuse?category=report-content&report=adelhoonar&report_content_url=https%3A%2F%2Fgithub.com%2Fjlevy%2Fthe-art-of-command-line%2Fissues%2F809&report_id=113869398&report_type=content)